lib/s3routes/routes.js

```diff
@@ -19,6 +19,8 @@ const routeMap = {
     OPTIONS: routeOPTIONS,
 };
 
+const OBJECT_KEY_BYTE_LIMIT = 915;
```

Comment:
Please move this to `constants.js`. I will help us customize these values easily rather than hunting them down in files.